python/fledge/services/core/service_registry/monitor.py

```diff
@@ -61,6 +74,34 @@ async def _sleep(self, sleep_time):
     async def _monitor_loop(self):
         """async Monitor loop to monitor registered services"""
         # check health of all micro-services every N seconds
+
+        '''
+        def start_storage():
+            with  open("/tmp/monitor.log", "w+") as file:
```

Comment:
What is this file used for? It looks like it is for your debugging purposes and should not be part of the final merge.